config.json

```diff
@@ -214,7 +214,12 @@
       "core": true,
       "difficulty": 4,
       "slug": "triangle",
-      "topics": null,
+      "topics": [
+      	"booleans",
+      	"equality",
+      	"logic",
+      	"mathematics"
```

Comment:
Great suggestions! :) The only ones I'm not sure about are `equality` and `logic`. Why do you think they're important to this exercise?

I would also add `exception_handling` since the tests expect an exception to be thrown :)

Also, a minor issue, all the topics should be indented two spaces in from `topics`, see the other exercise entries for examples :)